mbed_app.json

```diff
@@ -12,6 +12,9 @@
     "target_overrides": {
         "*": {
             "platform.stdio-convert-newlines": true
+        },
+        "DISCO_L475VG_IOT01A": {
+            "ism43362.provide-default": true
```

Comment:
Could we also do
```
ST DISCO_F413ZH board with integrated ISM43362 WiFi Inventek module.
```

Reply:
Yes, it's the same situation - no other connectivity option, so worth adding it to the default `mbed_app.json`.